public/modules/custom/grants_webform_import/src/Commands/WebformImportCommands.php

```diff
@@ -428,10 +428,8 @@ private function getWebformDataFromEndpoint() {
         'Authorization' => $authorizationHeader,
       ],
     ];
-    $response = $this->httpClient->get(
-      $url,
-      $options,
-    );
+    $request = $this->httpClient->request("GET", $url, $options);
```

Comment:
` [error]  TypeError: GuzzleHttp\Client::send(): Argument #1 ($request) must be of type Psr\Http\Message\RequestInterface, GuzzleHttp\Psr7\Response given, called in /app/public/modules/custom/grants_webform_import/src/Commands/WebformImportCommands.php on line 432 in GuzzleHttp\Client->send() (line 120 of /app/vendor/guzzlehttp/guzzle/src/Client.php) #0 /app/public/modules/custom/grants_webform_import/src/Commands/WebformImportCommands.php(432): GuzzleHttp\Client->send()
#1 /app/public/modules/custom/grants_webform_import/src/Commands/WebformImportCommands.php(331): Drupal\grants_webform_import\Commands\WebformImportCommands->getWebformDataFromEndpoint()
#2 [internal function]: Drupal\grants_webform_import\Commands\WebformImportCommands->importThirdPartySettingsViaApi()
#3 /app/vendor/consolidation/annotated-command/src/CommandProcessor.php(276): call_user_func_array()
#4 /app/vendor/consolidation/annotated-command/src/CommandProcessor.php(212): Consolidation\AnnotatedCommand\CommandProcessor->runCommandCallback()
#5 /app/vendor/consolidation/annotated-command/src/CommandProcessor.php(175): Consolidation\AnnotatedCommand\CommandProcessor->validateRunAndAlter()
#6 /app/vendor/consolidation/annotated-command/src/AnnotatedCommand.php(387): Consolidation\AnnotatedCommand\CommandProcessor->process()
#7 /app/vendor/symfony/console/Command/Command.php(326): Consolidation\AnnotatedCommand\AnnotatedCommand->execute()
#8 /app/vendor/symfony/console/Application.php(1096): Symfony\Component\Console\Command\Command->run()
#9 /app/vendor/symfony/console/Application.php(324): Symfony\Component\Console\Application->doRunCommand()
#10 /app/vendor/symfony/console/Application.php(175): Symfony\Component\Console\Application->doRun()
#11 /app/vendor/drush/drush/src/Runtime/Runtime.php(110): Symfony\Component\Console\Application->run()
#12 /app/vendor/drush/drush/src/Runtime/Runtime.php(40): Drush\Runtime\Runtime->doRun()
#13 /app/vendor/drush/drush/drush.php(139): Drush\Runtime\Runtime->run()
#14 /app/vendor/drush/drush/drush(4): require('...')
#15 /app/vendor/bin/drush(119): include('...')
#16 {main}.`

`TypeError: GuzzleHttp\Client::send(): Argument #1 ($request) must be of type Psr\Http\Message\RequestInterface, GuzzleHttp\Psr7\Response given, called in /app/public/modules/custom/grants_webform_import/src/Commands/WebformImportCommands.php on line 432 in /app/vendor/guzzlehttp/guzzle/src/Client.php on line 120 #0 /app/public/modules/custom/grants_webform_import/src/Commands/WebformImportCommands.php(432): GuzzleHttp\Client->send()
#1 /app/public/modules/custom/grants_webform_import/src/Commands/WebformImportCommands.php(331): Drupal\grants_webform_import\Commands\WebformImportCommands->getWebformDataFromEndpoint()`

Noi tulee siis ku ajaa drush gwia3 komennon. Eli ei näyttäs ihan täysin toimivan tuo requestjumppaus?